src/js/components/status/LiveStatusIndicator.jsx

```diff
@@ -1,25 +1,8 @@
-// @flow
-
 import React, {Component, PropTypes} from 'react';
 import moment from 'moment';
 import { StatusIndicator, decodeResultValue } from './StatusIndicator';
 
-const {number, string} = PropTypes;
-
-// TODO: avoid duplicating flow constants from StatusIndicator
-const validResultValues = {
-    success: 'success',
-    failure: 'failure',
-    running: 'running',
-    queued: 'queued',
-    unstable: 'unstable',
-    aborted: 'aborted',
-    not_built: 'not_built',
-    unknown: 'unknown'
-};
-
-// Enum type from const validResultValues
-export type Result = $Keys<typeof validResultValues>;
+const {bool, number, string} = PropTypes;
```

Comment:
Just to clarify, we want to use PropTypes.bool rather than importing it as a const?